Dockerfile.brb

```diff
@@ -19,4 +19,3 @@ RUN gulp && rm .gulp-changed-smart.json
 
 VOLUME /home/node/app/build
 CMD npm start
-
```

Comment:
files should end with one empty line, always!

Reply:
In my Dockerfile.brb is one empty line at the end...?